fix(service marketplace): fix detail images

Usmanfee opened the pull request:
## Description
Service marketplace and service marketplace detail page has hardcoded images are being displayed. The original image should be displayed incase of its existence otherwise default images would be displayed.

Changelog entry:

```
- **Service Marketplace**
  - replaced hardcoded images in service marketplace and detail [#1195](https://github.com/eclipse-tractusx/portal-frontend/issues/1195)
```

## Solution
There was one parameter of `leadPictureId` was missing in response of api which actually contains the image info so with leadPictureId and now by having `leadPictureId` we are displaying actual images.


## Issue
https://github.com/eclipse-tractusx/portal-frontend/issues/1195

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have updated CHANGELOG.md file

shubhamv-ss commented on src/components/pages/ServiceMarketplace/RecommendedServices.tsx:
```diff
@@ -73,7 +73,11 @@ export default function RecommendedServices({
+                  service?.leadPictureId
+                    ? `${getApiBase()}/api/administration/documents/${service.leadPictureId}`
+                    : `${getAssetBase()}/images/content/ServiceMarketplace.png`
+                }
```
This code is used in the below file. Consider converting it into a reusable function for efficiency.

shubhamv-ss commented on src/components/pages/ServiceMarketplace/ServicesElements.tsx:
```diff
@@ -81,7 +81,11 @@ export default function ServicesElements({
                       borderRadius={6}
                       imageAlt="App Card"
-                      imagePath={`${getAssetBase()}/images/content/ServiceMarketplace.png`}
+                      imagePath={
```
Try to put this in constant `/images/content/ServiceMarketplace.png`

shubhamv-ss reviewed (approved):
lgtm

typecastcloud commented on CHANGELOG.md:
```diff
@@ -95,6 +95,7 @@
 - **App Management**
   - fixed 400 Bad Request error due to search filter [#1057](https://github.com/eclipse-tractusx/portal-frontend/pull/1058)
   - added load more button app overview [#1009](https://github.com/eclipse-tractusx/portal-frontend/pull/1009)
+  - replaced hardcoded images in service marketplace and detail [#1195](https://github.com/eclipse-tractusx/portal-frontend/issues/1195)
```
This version was already released. Please move changelog entry to "unreleased".

oyo reviewed (changes requested):
Too much custom image loading logic that already has been implemented. See comments about how to use that. Also please update DEPENDENCIES.

evegufy reviewed (changes requested):
Check for build is failing.

Please also remove the changes to changelog and add a note in the PR description.

Usmanfee commented:
@evegufy  I have reverted the change of changelog file. Regarding build failure, failure is expected since I introduced new changes in shared component which is not yet merged and released so that is why it is failing with linting.

https://github.com/eclipse-tractusx/portal-shared-components/issues/356
this above PR first need to be merged and released in order to fix build failure for this PR.

If you have any other suggestion or work around to address this blocker then please let me know! Thanks

evegufy commented:
> @evegufy I have reverted the change of changelog file. Regarding build failure, failure is expected since I introduced new changes in shared component which is not yet merged and released so that is why it is failing with linting.
> 
> [eclipse-tractusx/portal-shared-components#356](https://github.com/eclipse-tractusx/portal-shared-components/issues/356) this above PR first need to be merged and released in order to fix build failure for this PR.
> 
> If you have any other suggestion or work around to address this blocker then please let me know! Thanks

Hi @Usmanfee ok understood, @oyo is working on the issue why the integration takes to long, just fyi https://github.com/eclipse-tractusx/portal-shared-components/issues/361

manojava-gk commented:
Resolve conflicts and fix pr checks issue

lavanya-bmw commented:
Please fix PR checks issue

Usmanfee commented:
@evegufy  @manojava-gk  can you please a have review now ? I have resolved conflicts and failed pipeline issue. Thanks

lavanya-bmw reviewed (approved):
lgtm

lavanya-bmw reviewed (changes requested):
Please resolve conflicts

ntruchsess reviewed (changes requested):
as there are new conflicts we cannot proceed. please resolve those new conflicts

manojava-gk reviewed (commented):
changes looks fine. but resolve conflicts

Usmanfee commented:
@lavanya-bmw  @manojava-gk  can you please check now ? I have resolved the merge conflicts. Thanks

manojava-gk reviewed (approved):
lftm

typecastcloud commented on yarn.lock:
```diff
@@ -7403,4 +7403,4 @@ yn@3.1.1:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
-  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
+  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
```
Please do not remove end of file new line.

saadanzari commented:
@saadanzari  to add review

saadanzari commented:
@Usmanfee pls fix the conflicts and then I can add my review

Usmanfee commented:
> @Usmanfee pls fix the conflicts and then I can add my review

@saadanzari  please review. Thanks

JordanGerada3 reviewed (approved):
LGTM!

evegufy commented:
> @Usmanfee you can merge this after the requested changes are approved by one of the reviewers

@saadanzari just to clarify, only committers can merge
